src/App.js

```diff
@@ -1,5 +1,78 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
+function printResult(sum){
+  return MissionUtils.Console.print(`결과 : ${sum}`);
+}
+
+function sumNumber(numberArr){
+  let sum = 0;
+  for(var number of numberArr) sum += number;
+  return sum;
```

Comment:
```suggestion
  let sum = 0;
  for(const number of numberArr) sum += number;
  return sum;
```
지금 방식으로 한다면 `for...of`문 내부에서는 `const` 키워드로 선언해도 무방할 것 같습니다!

함수형 프로그래밍을 사용하신다면 `reduce`로 선언적인 방식으로 코드를 작성하시는 것도 좋을 것 같아요🙃
```js
return numberArr.reduce((acc, cur) => acc + cur, 0);
```